src/studio-home/card-item/index.tsx

```diff
@@ -92,48 +87,27 @@ const CardItem: React.FC<Props> = ({
         )}
         subtitle={subtitle}
         actions={showActions && (
-          isPaginated ? (
-            <Dropdown>
-              <Dropdown.Toggle
-                as={IconButton}
-                iconAs={MoreHoriz}
-                variant="primary"
-                data-testid="toggle-dropdown"
-              />
-              <Dropdown.Menu>
-                {isShowRerunLink && (
-                  <Dropdown.Item
-                    as={Link}
-                    to={rerunLink ?? ''}
-                  >
-                    {messages.btnReRunText.defaultMessage}
-                  </Dropdown.Item>
-                )}
-                <Dropdown.Item href={lmsLink}>
-                  {intl.formatMessage(messages.viewLiveBtnText)}
-                </Dropdown.Item>
-              </Dropdown.Menu>
-            </Dropdown>
-          ) : (
-            <ActionRow>
-              {isShowRerunLink && (
-                <Hyperlink
-                  className="small"
-                  destination={trimSlashes(rerunLink ?? '')}
-                  key={`action-row-rerunLink-${courseKey}`}
-                >
-                  {intl.formatMessage(messages.btnReRunText)}
-                </Hyperlink>
-              )}
-              <Hyperlink
-                className="small ml-3"
-                destination={lmsLink ?? ''}
-                key={`action-row-lmsLink-${courseKey}`}
+        <Dropdown>
+          <Dropdown.Toggle
+            as={IconButton}
+            iconAs={MoreHoriz}
+            variant="primary"
+            data-testid="toggle-dropdown"
```

Comment:
As mentioned above, I think this needs an `aria-label`:
```suggestion
            aria-label={intl.formatMessage(messages. ... )}
```

See also https://github.com/openedx/paragon/issues/3616